libdnf/rpm/transaction.cpp

```diff
@@ -355,7 +356,7 @@ class Transaction::Impl {
         FD_t fd = Fopen(file_path.c_str(), "r.ufdio");
 
         if (!fd) {
-            throw Exception("read_pkg_header: Can't open file: " + file_path);
+            throw TransactionError(M_("read_pkg_header: Cannot open file \"{}\": {}"), file_path, Fstrerror(fd));
```

Comment:
Ok, if changing these, please remove the `read_pkg_header`. I know it might be the clearest for a developer, but it has no relevance for the user. Use something like this instead:
> Failed to read package header, cannot open file \\"{}\\": {}

Reply:
Ok.